content/shmem_request_test.tex

```diff
@@ -0,0 +1,29 @@
+\apisummary{
+    The request test routine provides a method for querying the status
+    of the request.
+}
+
+\begin{apidefinition}
+
+\begin{Csynopsis}
+void shmem_request_test(shmem_request_t *req, int *status);
```

Comment:
I have not defined any constants. All I say that the status will be 0 if the request is complete and non zero otherwise.